src/js/components/summary_column_header.coffee

```diff
@@ -0,0 +1,45 @@
+React      = require 'react'
+_          = require 'lodash'
+
+ReactRouter = require 'react-router'
+Link        = React.createFactory(ReactRouter.Link)
+{div, th}   = React.DOM
+
+SummaryColumnHeader = React.createClass
+
+  clickPage: () ->
+    if @props.setPageId
+      @props.setPageId(@props.pageId)
+    else
+      console.log @props.pageId
+
+  clickQuestion: (question) ->
+    if @props.clickQuestion
+      @props.clickQuestion(@props.pageId, question)
+    else
+      console.log "Question clicked #{@props.pageId}"
+
+  render: ->
+    pageName =  _.truncate(@props.name, {length: 20})
```

Comment:
Instead of truncating you could use `text-overflow: ellipsis` in the css (https://www.w3schools.com/cssref/css3_pr_text-overflow.asp)